hydradx/notebooks/Misc/load_omnipool_balances.ipynb

```diff
@@ -0,0 +1,468 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "7e3246da-b5f4-45e1-9aad-56772bff032b",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "C:\\Users\\jepid\\PycharmProjects\\HydraDX-simulations\\hydradx\\notebooks\\Misc\n"
+     ]
+    }
+   ],
+   "source": [
+    "from matplotlib import pyplot as plt\n",
+    "import sys\n",
+    "import requests\n",
+    "import base64\n",
+    "import json\n",
+    "from pprint import pprint\n",
+    "from dotenv import load_dotenv\n",
+    "import os\n",
+    "import time\n",
+    "\n",
+    "sys.path.append('../..')\n",
+    "load_dotenv()\n",
+    "username = os.getenv('SQLPAD_USERNAME')\n",
+    "password = os.getenv('PASSWORD')\n",
+    "print(os.getcwd())\n",
+    "print(username, password)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "2585953b-2472-4b7b-9547-78a9f95499d6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "chunk_size = 10000\n",
+    "chunks_per_file = 100\n",
+    "\n",
+    "# Encode the username and password in Base64\n",
+    "credentials = f\"{username}:{password}\"\n",
+    "encoded_credentials = base64.b64encode(credentials.encode('utf-8')).decode('utf-8')\n",
+    "\n",
+    "headers = {\n",
+    "    'Authorization': f'Basic {encoded_credentials}',\n",
```

Comment:
I don't believe this is encrypted, correct? So we should absolutely be certain we are also not printing the "headers" object anywhere?